src/haz3lweb/exercises/SyntaxTest.re

```diff
@@ -201,6 +213,7 @@ let rec var_mention = (name: string, uexp: Exp.t): bool => {
   | BuiltinFun(_) => false
   | Cast(d, _, _) => var_mention(name, d)
   | Ap(_, u1, u2)
+  | Dot(u1, u2)
```

Comment:
check to make sure label positions are not being recursed into for variable mentions / recursive functions / etc.

Reply:
@cyrus- since we don't have the label sort introduced, the variables in label position are still being evaluated:

![Screenshot From 2025-02-10 10-24-37](https://github.com/user-attachments/assets/f993d061-3b1b-44ef-8311-bca2666964e8)


I can try to make these indet, but it's not clear to me that it's preferable to stop dynamics for the expression. If we still allow evaluation I think we want to continue to recurse into the "label position" in SyntaxTest.

Reply:
This seems somewhat related to decisions for now evaluation should proceed in non-empty holes https://github.com/hazelgrove/hazel/issues/1497#issuecomment-2640785436.

Reply:
I've put everything in label position in multiholes if it's not a label.